gui/static/js/components/blockly.js

```diff
@@ -42,11 +60,66 @@ export default class Blockly extends React.Component {
         document.getElementById('data').value = window.Blockly.Python.workspaceToCode(this.workspace);
     }
 
+     /* DOWNLOAD FUNCTION
+       Allows users to download raw code as a file. Users must
+       manually input file name and file ext.
+    */
+    download(event){
+        console.log("download listener");
```

Comment:
Remove this print statement. It seems like it was something that was necessary for debugging but not for the actual users.